openmdao_book/features/core_features/working_with_components/continuous_variables.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<p>not clear what's going on here with <code>print</code> version above versus <code>assert</code> versions below...</p><p>see previous question about whether we are going to automatically transform the asserts to prints...</p><p>(also subsequent occurrences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>Because we aren't automatically converting asserts to print, we put the assert in a separate cell and then replace the asserts with prints to emulate the old docs</p>

Reply:
@sryan : One other note on this: the blocks with asserts in them are hidden using the tags `remove-input` and `remove-output`